authors/apps/articles/renderers.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework.renderers import JSONRenderer
+import json
+
+
+class ArticleJSONRenderer(JSONRenderer):
```

Comment:
Hello @Bakley ,
Since you are not using this class and the file at large, you could do away with them.

Reply:
Okay doing away with it